unittests/POSIX/CMakeLists.txt

```diff
@@ -49,4 +49,4 @@ add_custom_target(
   posix_tests
   WORKING_DIRECTORY "${CMAKE_BINARY_DIR}"
   USES_TERMINAL
-  COMMAND "ctest" "-j${CORES}" "-VV" "-R" "\.*.posix")
+  COMMAND "ctest" "--timeout" "302" "-j${CORES}" "-R" "\.*.posix")
```

Comment:
Why move timeout here and remove it from the runner file?

Reply:
See reply above

Reply:
You can also override per-test via cmake, so there's no need at all for us to be doing timeout